src/test/java/com/epam/qalab/DeliverToTest.java

```diff
@@ -0,0 +1,101 @@
+package com.epam.qalab;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeliverToTest {
+    WebDriver driver;
+
+    @BeforeTest
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        WebDriverManager.chromedriver().browserVersion("97").setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.amazon.com/");
+    }
+
+    @Test
+    public void verifyZipCodeUpdateTest() {
+        WebElement oldLocation = driver.findElement(By.id("glow-ingress-line2"));
+
+        WebElement deliverToButton = driver.findElement(By.id("nav-global-location-popover-link"));
+        deliverToButton.click();
+
+        WebElement zipCodeField = new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions.visibilityOfElementLocated(By.id("GLUXZipUpdateInput")));
```

Comment:
small note, this line can be split into two lines, then we don't need to scroll horizontally to see what it contains,
this also refers to other long lines